CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 ### Added
+- OxyPlot.Series.HeatMapSeries.GetValueForLabels (#256)
```

Comment:
Both GetValue and GetValueForLabel are "private static double" methods. The intention is to make them similar. 

GetValueForLabels should be renamed to GetValueForLabels and that will be reflected when I re-sync the edits.

GetValueForLabel is necessary because GetValue returns double.NaN at some boundaries. These boundaries are almost non-detectable in a continuous dimension (that is to say, an interpolated dimension) used for mouse movements and a color spectrum. Unfortunately that boundary is at whole number coordinates. Eliminate the possibility of a calculated double.NaN at some boundaries by introducing GetValueForLabel. Double.NaN is still possible in GetValueForLabel but those values are not calculated, just read from memory. Since RenderLabels always uses integer coordinates, GetValueForLabel receives integers for type-safety. GetValueForLabel is simple (more compute efficient) than GetValue.